content/workshops/example/workshop_content.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Workshop Content
+workshops: example
+workshop_weight: 30
+layout: lab
+---
+
+## SETTING EXPECTATIONS FOR YOUR AUDIENCE
```

Comment:
Use title case for your headings instead of all caps to match style of other labs in the example workshop.

I'll only put the comment here instead of copy/pasting it for each one but check each of your files for any places that have all caps for titles.